pymc/distributions/continuous.py

```diff
@@ -524,11 +525,11 @@ class Normal(Continuous):
 
     Parameters
     ----------
-    mu: float
-        Mean.
-    sigma: float
+    mu : float
+        Mean. Defaults to 0.
+    sigma : float
```

Comment:
```suggestion
    sigma : float, optional
```

here the default is not immediate as it depends on the value of tau too, so it should use optional only with the default on the description. In this case the description should mention that when no value is passed for tau nor sigma, sigma defaults to one